src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -493,8 +493,9 @@ static int do_map()
     }
 
     if (forker.is_parent()) {
+      global_init_postfork_start(g_ceph_context);
```

Comment:
maybe we can move this line out above `if (forker.is_parent())` block, so it always gets called after `fork()`? and remove the call at line 636 in the `if (g_conf->daemonize)` block. so both child and parent's logger can be started, and can be stopped properly by the dtor of `g_ceph_context`?